scripts/open-in-repl.js

```diff
@@ -0,0 +1,33 @@
+/* global jQuery */
+
+(function ($) {
+    'use strict';
+
+    var presets = [ "es2015", "stage-0", "react" ];
+
+    var selector = '.language-js,.language-javascript';
+    var clipboardIcon = '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" width="14" height="15"><path fill="currentColor" d="M11.5 7c-.276 0-.5-.224-.5-.5C11 5.122 9.878 4 8.5 4c-.276 0-.5-.224-.5-.5s.224-.5.5-.5C9.878 3 11 1.878 11 .5c0-.276.224-.5.5-.5s.5.224.5.5C12 1.878 13.122 3 14.5 3c.276 0 .5.224.5.5s-.224.5-.5.5C13.122 4 12 5.122 12 6.5c0 .276-.224.5-.5.5zm-1.199-3.5c.49.296.903.708 1.199 1.199.296-.49.708-.903 1.199-1.199-.49-.296-.903-.708-1.199-1.199-.296.49-.708.903-1.199 1.199zM1.5 10c-.276 0-.5-.224-.5-.5S.776 9 .5 9 0 8.776 0 8.5.224 8 .5 8s.5-.224.5-.5.224-.5.5-.5.5.224.5.5.224.5.5.5.5.224.5.5-.224.5-.5.5-.5.224-.5.5-.224.5-.5.5zm16.647 5.939L7.561 5.353c-.283-.283-.659-.438-1.061-.438s-.778.156-1.061.438l-.586.586c-.283.283-.438.659-.438 1.061s.156.778.438 1.061l10.586 10.586c.283.283.659.438 1.061.438s.778-.156 1.061-.438l.586-.586c.283-.283.438-.659.438-1.061s-.156-.778-.438-1.061zM5.561 6.646l.586-.586c.094-.094.219-.145.354-.145s.26.052.354.145l1.439 1.439-1.293 1.293-1.439-1.439c-.195-.195-.195-.512 0-.707zm11.878 10.708l-.586.586c-.094.094-.219.145-.353.145s-.26-.052-.353-.145L7.708 9.501l1.293-1.293 8.439 8.439c.195.195.195.512 0 .707zM3.5 5c-.276 0-.5-.224-.5-.5C3 3.673 2.327 3 1.5 3c-.276 0-.5-.224-.5-.5s.224-.5.5-.5C2.327 2 3 1.327 3 .5c0-.276.224-.5.5-.5s.5.224.5.5C4 1.327 4.673 2 5.5 2c.276 0 .5.224.5.5s-.224.5-.5.5C4.673 3 4 3.673 4 4.5c0 .276-.224.5-.5.5zm-.502-2.5c.19.143.359.312.502.502.143-.19.312-.359.502-.502-.19-.143-.359-.312-.502-.502-.143.19-.312.359-.502.502zM3.5 15c-.276 0-.5-.224-.5-.5 0-.827-.673-1.5-1.5-1.5-.276 0-.5-.224-.5-.5s.224-.5.5-.5c.827 0 1.5-.673 1.5-1.5 0-.276.224-.5.5-.5s.5.224.5.5c0 .827.673 1.5 1.5 1.5.276 0 .5.224.5.5s-.224.5-.5.5c-.827 0-1.5.673-1.5 1.5 0 .276-.224.5-.5.5zm-.502-2.5c.19.143.359.312.502.502.143-.19.312-.359.502-.502-.19-.143-.359-.312-.502-.502-.143.19-.312.359-.502.502z"/></svg>';
+
+    var button = $(
+      '<button class="hidden-xs btn btn-link btn-icon btn-open-in-repl" aria-label="Try in REPL">'
+      + clipboardIcon
+      + '<strong class="btn-icon__label small">Try</strong>'
+      + '</button>'
+    );
+
+    button.on('click', function() {
+      var $code = $(this).parent().find('code');
+
+      var url = '/repl/#'
+              + '?babili=false'
+              + '&evaluate=false'
```

Comment:
Auto-evaluation is disabled since some examples don't produce an output or can't run.